src/utils/validation/registration/registrationValidation.ts

```diff
@@ -21,8 +21,12 @@ export const registrationValidationSchema = Yup.object().shape({
     npiSubjectHeading: Yup.string(),
     date: Yup.object().shape({
       year: Yup.number().required(ErrorMessage.REQUIRED),
-      month: Yup.number(),
-      day: Yup.number(),
+      month: Yup.number()
+        .nullable()
+        .transform((value: string, originalValue: string) => (originalValue === '' ? null : value)),
+      day: Yup.number()
+        .nullable()
+        .transform((value: string, originalValue: string) => (originalValue === '' ? null : value)),
```

Comment:
trenger kanskje ikke originalValue lenger uten trim?
```suggestion
        .transform((value: string) => (value=== '' ? null : value)),
```

Reply:
Jo. Se kommentaren min over 👆🏼

Reply:
hm, skjønner ikke helt det der as :p

Reply:
Ikke jeg heller egentlig. Dårlig eksempel på yup-siden. Men orker ikke å bruke mer tid på denne nå.